### ISSUE TITLE
TODO: Port to the 3DS

### ISSUE THREAD

**Issue by Possum** [chaoskagami/vndc#1]
The README mentions one of the goals is to target 3ds CFW. Is there any implementation of this yet, and if so, how can I build it?

**Comment by chaoskagami** [comment]
No, there's a reason it's mentioned as a goal. I'm not able to work on this as actively, and realistically it was rather poorly implemented in the first place (so it desperately needs refactoring.)

It's not currently buildable for the 3DS, and you shouldn't expect it to be anytime soon. I need to rewrite the SDL2 code onto graphics commands for ctrulib, and reimplement my sound layer against DSP.

tl;dr not anytime soon, and I never actually said anywhere it was supported yet, just that I'd like to someday.

I'll leave this open and change the title as a big fat notice.
